### DISCUSSION TITLE
DarkTheme does not working for fiill style

### DISCUSSION THREAD

**Discussion by balex777** [tailwindlabs/tailwindcss#5558] [Help]
Hello, I try to use dark:fill-white but it's dose not working for svg. In code I see `class="fill-warrningStrong dark:fill-white"`
My config include this

```
...
  darkMode: 'class', // or 'media' or 'class'

  theme: {

    fill: theme => ({

      white: theme('colors.white'),

      warrningStrong: theme('colors.yellow.warrningStrong')

    }),
...
```

How to fix it?

**Comment by simonswiss** [comment]
Hey! Can you reproduce the issue in a [Play](https://play.tailwindcss.com) example, so it's a little easier to help and explain what the issue is?

**Reply by balex777** [reply]
Hello, I fix it. Just need to include no variants in config 
```
  variants: {
    extend: {
      fill: ['dark'],
    },
  },
```
Thanks
